neural_compressor/onnxrt/algorithms/smooth_quant/smooth_quant.py

```diff
@@ -0,0 +1,653 @@
+#
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023 Intel Corporation

```

Comment:
license year should be updated to 2023 - 2024 for all modified files

Reply:
We'd better update the license template utilized by pre-commit to address this common issue.